ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4159,6 +4159,91 @@ AHU_CHWSC_ECON_FDPM_HWSC_SFSS_SFVSC_SSPC_SDHC_RTM:
   opt_uses:
   - smoke_alarm
 
+AHU_SFSS4X_SFVSC4X_RFSS3X_RFVSC3X_CHWSC_HWSC_ECON_EDBPC_SARC_SSPC_RD_OAFMC_EFM:
+  guid: "187ad041-6b1e-4ee5-aabb-69dc8c287bdd"
+  description: "Cooling and heating AHU with 4 supply and 3 return fans, supply air reset control, 
+                building and supply static pressure control, return damper flow control and exhaust flow monitoring."
+  is_canonical: true
+  implements:
+  - AHU
+  - SFSS4X
+  - SFVSC4X
+  - RFSS3X
+  - RFVSC3X
+  - CHWSC
+  - HWSC
+  - ECON
+  - EDBPC
+  - SARC
+  - SSPC
+  - RD
+  - OAFMC
+  - EFM
+  opt_uses:
+  - smoke_alarm
+
+AHU_SFSS4X_SFVSC4X_RFSS3X_RFVSC3X_CHWSC_HWSC_ECON_EDBPC_SARC_SSPC_RD_OAFMC:
+  guid: "dca00b39-6816-4d59-b3b6-2b3a429edf8b"
+  description: "Cooling and heating AHU with 4 supply and 3 return fans, supply air reset control, 
+                building and supply static pressure control and return damper flow control."
+  is_canonical: true
+  implements:
+  - AHU
+  - SFSS4X
+  - SFVSC4X
+  - RFSS3X
+  - RFVSC3X
+  - CHWSC
+  - HWSC
+  - ECON
+  - EDBPC
+  - SARC
+  - SSPC
+  - RD
+  - OAFMC
+  opt_uses:
+  - smoke_alarm
+
+AHU_SFSS2X_SFVSC2X_RFSS2X_RFVSC2X_CHWSC_HWSC_ECON_EDBPC_SARC_SSPC_RD_OAFMC:
+  guid: "5a3034e7-0a17-416c-b974-55072d52d3e4"
+  description: "Cooling and heating AHU with 2 supply and 2 return fans, supply air reset control, 
```

Comment:
nit: please remove carriage return in description text block (description text should just be on a single line)

Reply:
fixed